services/uwsgi.ini

```diff
@@ -16,9 +18,26 @@ chmod-socket = 666
 logto = /var/log/uwsgi/%n.log
 log-format = %(addr) - - [%(ltime)] "%(method) %(uri) %(proto)" %(status) %(size) "%(referer)" "%(uagent)" host_hdr=%(host) req_time_elapsed=%(msecs)
 
-processes = 6
+; max number of workers; %k is the number of detected CPU cores
+processes = %(%k + 1)
 threads = 2
+; forcefully kill workers after 300s
 harakiri = 300
 lazy-apps = true
-touch-reload = %(base)/uwsgi.ini
+touch-workers-reload = %(base)/uwsgi.ini
```

Comment:
I'm not sure if `touch-reload` worked before, but `touch-workers-reload` is the suggested command to use with `lazy-apps`.